Frey/Dockerfile

```diff
@@ -130,36 +114,23 @@ RUN ln -sf /dev/stdout /var/log/nginx/access.log \
     && ln -sf /dev/stderr /var/log/php81/error.log
 
 # Expose ports
-EXPOSE 4001 4002 4003 4004 4007 $DB_PORT
+EXPOSE 4004
 
 # copy entry point
 COPY entrypoint.sh /entrypoint.sh
 RUN chmod 755 /entrypoint.sh
```

Comment:
It would be useful to add:
RUN dos2unix /entrypoint.sh

If run under WSL windows encoding may corrupt file